File: Farm_keeper/Farm_keeper/src/Model/Dog.java
package Model;

import java.text.ParseException;
import java.util.Date;

import javax.xml.crypto.Data;

public class Dog extends Pet {

    /**
     * @param name
     * @param command
     * @param bday
     * @param class_name
     * @throws ParseException
     */
    public Dog(String name, String command, String bday) throws ParseException {

        this.name = name;
        this.command = command;
        this.bday = bday;
        this.class_name = "Dog";
        new_entry(name, command, bday, class_name);


    }

}

File: Python Farm_keeper/Model/Dog.py
from Model.Pet import Pet


class Dog(Pet):

    def __init__(self, name: str, command: str, bday: str):
        super().__init__(name, command, bday)
        self.__name = name
        self.__command = command
        self.__bday = bday
        self.class_name = "Dog"



